nomad/structs/structs.go

```diff
@@ -4156,17 +4156,19 @@ const (
 	// JobMinPriority is the minimum allowed priority
 	JobMinPriority = 1
 
-	// JobDefaultPriority is the default priority if not
-	// not specified.
+	// JobDefaultPriority is the default priority if not specified.
 	JobDefaultPriority = 50
 
+	// JobDefaultMaxPriority is the default maximum allowed priority
+	JobDefaultMaxPriority = 100
+
 	// JobMaxPriority is the maximum allowed priority
```

Comment:
Just to make the purpose of this const clear:

```suggestion
	// JobMaxPriority is the maximum allowed configuration value for maximum job priority
```